tests/integrations_plugin_wordpress_seo/_bootstrap.php

```diff
@@ -0,0 +1,9 @@
+<?php
+// Let's make sure to set rewrite rules.
+global $wp_rewrite;
+$wp_rewrite->permalink_structure = '/%postname%/';
+$wp_rewrite->rewrite_rules();
+
+update_option( 'theme', 'twentytwenty' );
```

Comment:
Don't do this any more - especially since you're not doing snapshots.
default WP installs don't include 2020